Program.cs

```diff
@@ -40,13 +43,14 @@ static void Run(Args a)
                 metrics = proxy.Materialize(a.DropDestination).Result;
                 Console.WriteLine($"Finished in {sw.Elapsed}");
                 props["success"] = "True";
-                
+                return 0;

```

Comment:
always felt that throwing the exception was the right thing to do ratehr thand catching and returning error codes. It's simpler and I think you still get a non zero return code is that not right?

Reply:
My sample app tells me you're right, but I don't see much documentation on what the return code would be. I agree that you generally shouldn't change exceptions to return codes, but I think the exception to that would be when you cross managed to native code, which we sort of do when we exit main. So I would personally prefer that it be explicit about the return value in the main function.